packages/wdio-cli/tests/commands/__snapshots__/config.test.ts.snap

```diff
@@ -39,7 +39,7 @@ For for information on TypeScript integration check out: https://webdriver.io/do
     "
 Configuration file was created successfully!
 To run your tests, execute:
-$ npx wdio run wdio.conf.js
+$ npx wdio run wdio.conf.ts
```

Comment:
why are these now `.ts`? is this because the project has a `tsconfig.json` file? if so we should remove that dependency

Reply:
No it is because the unit test is simulating a configuration with TypeScript as compiler.

Reply:
@praveendvd however I don't believe they should be all TypeScript. Let's mock `hasFile` so that it only returns `true` when running our TypeScript unit test.

Reply:
@christian-bromann  Thats the success message , there i am not checking if tsconfig.json is there.  Typescript auto generated project doesn't have wdio.config.js . should i change message to ` npx wdio run wdio.conf.ts or  npx wdio run wdio.conf.js`

Reply:
@christian-bromann  i will set hsFIle to return true for actual tests.

Reply:
Nevermind this, seems the unit test all are already like this.